package ftl.run.exception

const val SUCCESS = 0
const val GENERAL_FAILURE = 1
const val CONFIGURATION_FAIL = 2
const val NOT_PASSED = 10
const val UNEXPECTED_ERROR = 15
const val INCOMPATIBLE_TEST_DIMENSION = 18
const val CANCELED_BY_USER = 19
const val INFRASTRUCTURE_ERROR = 20
